specs/waku/waku-1.md

```diff
@@ -78,13 +79,24 @@ This protocol needs to advertise the `waku/1` [capability](https://ethereum.gitb
 
 In Whisper, envelopes are gossiped between peers. Whisper is a form of rumor-mongering protocol that works by flooding to its connected peers based on some factors. Envelopes are eligible for retransmission until their TTL expires. A node SHOULD relay envelopes to all connected nodes if an envelope matches their PoW and bloom filter settings. If a node works in light mode, it MAY choose not to forward envelopes. A node MUST NOT send expired envelopes, unless the envelopes are sent as a [mailserver](./mailserver.md) response. A node SHOULD NOT send an envelope to a peer that it has already sent before.
 
+### Maximum Packet Size
+
+Nodes SHOULD limit the maximum size of both packets and envelopes. If a packet exceeds its limit it MUST be dropped.
+
+- **RLPx Packet** - This size MUST be checked before a message is decoded.
+- **Waku Envelope** - Each envelope contained in an RLPx packet MUST then separately be checked against the maximum envelope size.
+
+Clients are free to use their own packet sizes. The default values are `10mb` for the RLPx Packet and `1mb` for a Waku envelope.
```

Comment:
I would considerably lower the size.
`10mb` is quite a lot, and `rlp` encoding/decoding is fairly expensive, `1.5mb` should be already plenty and it's lower risk for low-powered devices.

Reply:
Agree

Reply:
@cammellos what size would you allow for packets?

Reply:
1.5 maybe? 2mb if we want to have a round number